modules/dynamic-plugins/ref-example-third-party-plugin-installation.adoc

```diff
@@ -0,0 +1,77 @@
+[id="ref-example-third-party-plugin-installation_{context}"]
+= Example of installing a third-party plugin in {product}
+
+This section provides step-by-step instructions for integrating the link:https://github.com/backstage/community-plugins/tree/main/workspaces/todo/plugins[Todo plugin] into your {product-short}.
+
+. *Obtain the third-party plugin source code*: Clone the plugins repository and navigate to the link:https://github.com/backstage/community-plugins/tree/main/workspaces/todo/plugins[Todo plugin] directory:
++
+--
+.Example commands to obtain the third-party plugin source code
```

Comment:
Do we need this "Example commands ..." before every code block here? 
This whole document is called Example, it should be clear that the commands here are examples